WalletWasabi.Backend/Data/DesignTimeDbContextFactoryBase.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using Microsoft.Extensions.Configuration;
+
+namespace WalletWasabi.Backend.Data
+{
+	public abstract class DesignTimeDbContextFactoryBase<TContext> :
+		IDesignTimeDbContextFactory<TContext> where TContext : DbContext
+	{
+
+		public TContext CreateDbContext(string[] args)
+		{
+			return Create(
+				Directory.GetCurrentDirectory(),
+				Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT"));
+		}
+
+		protected abstract TContext CreateNewInstance(
+			DbContextOptions<TContext> options);
+
+		public TContext Create()
+		{
+			var environmentName =
+				Environment.GetEnvironmentVariable(
+					"ASPNETCORE_ENVIRONMENT");
+
+			var basePath = AppContext.BaseDirectory;
+
+			return Create(basePath, environmentName);
+		}
+
+		private TContext Create(string basePath, string environmentName)
```

Comment:
Replace body of this whole method with `return Create("User ID=postgres;Host=127.0.0.1;Port=65466;Database=doesntmatterbecauseitisnotactuallyused;");`

Reply:
just get rid of basePath and envName?